### ISSUE TITLE
Parameter not included when using swift style commenting with obj-c

### ISSUE THREAD

**Issue by awgeorge** [realm/jazzy#909]
While compiling obj-c using swift doc headers, while other keywords work, eg: `- returns:`, `- see: ` etc. Parameters will only appear using the `@param` syntax.

Does not appear
```
- parameter str:   The string to repeat.
```

Appears as a bullet point, including `param`
```
- param str:   The string to repeat.
```

Appears as a parameter
```
@param  The string to repeat.
```

Does not appear
```
- parameter str:   The string to repeat.
- see: nothing
```

Appears as standard text above the `see callout`
```
- param str:   The string to repeat.
- see: nothing
```

Appears as a bullet point
```
- param str:   The string to repeat.
- returns: nothing
```

So the `parameter` keyword is being parsed, but not being outputted in the documentation,

**Comment by johnfairh** [comment]
Right, currently you have to use the `@param` syntax in Obj-C mode.  Not done much testing on non-HeaderDoc -- latest Xcode 'generate-doc-comment' still generates HeaderDoc syntax.
